saskatoon/harvest/urls.py

```diff
@@ -84,9 +84,9 @@
          views.TemplateView.as_view(template_name='app/property_thanks.html'),
          name='property-thanks'),
 
-    path(r'stats/',
-         views.TemplateView.as_view(template_name='app/stats.html'),
-         name='statistics'),
+    re_path(r'^stats/(?P<season>\w+)/$', api.StatsView.as_view(),name='statistics'),
```

Comment:
here you wouldn't need to define a slug manually, REST filter can handle that for you